### DISCUSSION TITLE
Scrolling in the Explore page

### DISCUSSION THREAD

**Discussion by DriesDeMaeseneer** [community/community#146446] [Mobile] [answered]
I have noticed recently that I now have to scroll down a whole page to view the last update in the Explore page.

What is the reason for this change because it's quite inconvenient.

**Comment by stevepopovich** [comment]
Hi @DriesDeMaeseneer not sure I am understanding what you mean by this. Can you give me a screenshot or example?

**Reply by kickingvegas** [reply]
Sort order changed from date descending to date ascending. It is a very annoying regression.

**Reply by samadpls** [reply]
Yes, I'm experiencing the same issue. it's been two days and the problem remains unresolved. It's quite frustrating.

**Reply by NanotechPikachu** [reply]
Same here since around 2 days ago. It's nerve wrecking. Plz do fix

**Reply by Giorgi** [reply]
![Screenshot_20241207_125956_GitHub.png](https://github.com/user-attachments/assets/45791207-5e3b-4a34-a6e2-707e143cc5e3)

Here is a screenshot. It shows old updates at the top instead of showing recent items.

**Reply by michaelabechynova** [reply]
> Hi @DriesDeMaeseneer not sure I am understanding what you mean by this. Can you give me a screenshot or example?

-

**Reply by DriesDeMaeseneer** [reply]
> ![Screenshot_20241207_125956_GitHub.png](https://github.com/user-attachments/assets/45791207-5e3b-4a34-a6e2-707e143cc5e3)
> 
> Here is a screenshot. It shows old updates at the top instead of showing recent items.

Giorgi has posted a screenshot that  perfectly resembles the issue.

I have to scroll a month of activity before I reach the bottom due to the activities being sorted in an ascending way.

**Reply by Giorgi** [reply]
Looks like it's fixed now.

**Reply by NanotechPikachu** [reply]
Yeah, so it seems.

**Reply by stevepopovich** [reply, marked as answer]
Sorry about that folks, yep this was our bad, and things should be fixed now. Appreciate you all writing in and sorry for the inconvenience!
